### DISCUSSION TITLE
i need  php version  8.2.12

### DISCUSSION THREAD

**Discussion by DerpyCranel** [leokhoa/laragon#729] [General]
Where can I download php version 8.2.12 to use with laragon?

**Comment by Pen-y-Fan** [comment]
See my blog [How to update the PHP version in Laragon](https://pen-y-fan.github.io/2023/01/15/how-to-update-the-php-version-in-laragon/) for details.

**Reply by Antmans0000** [reply]
hello ! mine do not work,
can you help please ?
![image](https://github.com/user-attachments/assets/c1fd2737-b31d-4b80-a03e-ad103bb0d2ed)

**Reply by Pen-y-Fan** [reply]
The PHP directory doesn't look right, /php-8.3.9.tar/ should be like /php-8.3.8-nts-Win32-vs16-x64/

Are you using a matched version of PHP and Apache? The both need to be the same architecture.

See my blog [How to update the PHP version in Laragon](https://pen-y-fan.github.io/2023/01/15/how-to-update-the-php-version-in-laragon/) for more details. You need to download the zip file for Windows, unzip it and add it to the PHP directory.

**Comment by luisAntonioLAGS** [comment]
> Where can I download php version 8.2.12 to use with laragon?

Check this link, it is the repository of php versions for windows.
https://windows.php.net/downloads/releases/archives/
